src/test/java/kitchenpos/steps/IntegrationTestSteps.java

```diff
@@ -0,0 +1,149 @@
+        ExtractableResponse<Response> response = given().log().all()
+                                                        .contentType(ContentType.JSON)
+                                                        .body(menu)
+                                                        .when()
+                                                        .post(MENU_API)
+                                                        .then().log().all()
+                                                        .extract();
+
+        sharedContext.setResponse(response);
+    }
+
+    public void createProduct(Product product) {
+        ExtractableResponse<Response> response = given().log().all()
+                                                        .contentType(ContentType.JSON)
+                                                        .body(product)
+                                                        .when()
+                                                        .post(PRODUCT_API)
+                                                        .then().log().all()
+                                                        .extract();
+
+        sharedContext.setResponse(response);
+    }
+
+    public void createTable(OrderTable orderTable) {
+        ExtractableResponse<Response> response = given().log().all()
+                                                        .contentType(ContentType.JSON)
+                                                        .body(orderTable)
+                                                        .when()
+                                                        .post(ORDER_TABLE_API)
+                                                        .then().log().all()
+                                                        .extract();
+
+        sharedContext.setResponse(response);
+    }
+
+    public void changeNumberOfGuests(OrderTable orderTable, int number) {
+        orderTable.setNumberOfGuests(number);
+        ExtractableResponse<Response> response = given().log().all()
+                                                        .contentType(ContentType.JSON)
+                                                        .body(orderTable)
+                                                        .when()
+                                                        .put(ORDER_TABLE_API + "/"
+                                                            + orderTable.getId()
+                                                            + "/number-of-guests")
+                                                        .then().log().all()
+                                                        .extract();
+
+        sharedContext.setResponse(response);
+    }
+
+    public OrderTable changeEmpty(Long tableId, OrderTable table) {
+        return given().log().all()
+                      .contentType(ContentType.JSON)
+                      .body(table)
+                      .when()
+                      .put(ORDER_TABLE_API + "/" + tableId + "/empty")
```

Comment:
```suggestion
                      .put(ORDER_TABLE_API + "/{tableId}/empty", tableId)
```
저는 개인적으로 concat하는 것 보다 위와 같은 방식이 더 깔끔한 것 같더라구요!
이 부분은 선택적으로 반영해주시면 됩니다.

Reply:
오 이렇게 할 수 있군요! bb